docs/source/examplecleanup.rst

```diff
@@ -6,55 +6,106 @@ End-to-End Example
 
 An End-to-End example walkthrough of a tartufo scan and the process of purging the dirty evil passwords that somehow ended up in your code commits. We will use an additional tool: ``BFG`` (https://rtyley.github.io/bfg-repo-cleaner/, more on this later!). 
 
-#. Use ``tartufo`` to scan your repository and find any secrets in its history!
 
-   Use what you've learned to scan your repo!
+#. OPTIONAL Development only: Setup poetry if you want to use the most recent non-released build from github (may not be stable)
+
+   This project uses `Poetry`_ to manage its dependencies and do a lot of the heavy lifting. So you'll need to clone the tartufo repo and setup poetry!
+
+
+   .. code-block:: console
+
+      git clone git@github.com:godaddy/tartufo.git
+      
+   Development Use Only Poetry Setup: :doc:`CONTRIBUTING` setting-up-a-development-environment
+
+
+#. Clone your repo!
+
+   Select and clone the repo you want to run tartufo on
 
    .. code-block:: console
 
-      # Run Tartufo on your repo:
+      # Clone your repo, variables used later:
       GITHUBPROJECT="yourproject"
-      GITHUBREPO="yourrepo.git"
+      GITHUBREPO="myrepo.git"
       GITHUBADDRESS="github.com"
-      tartufo git@${GITHUBADDRESS}:${GITHUBPROJECT}/${GITHUBREPO}
-      # this will spit out a bunch of stuff and end with something like: Results have been saved in /tmp/tmp_jm3hyne
-      TARTUFOSCANPATH="/tmp/tmp_jm3hyne"
-      # this is the number of individual bad password hits tartufo has found:
-      ls ${TARTUFOSCANPATH} | wc -l
+      git clone --mirror git@${GITHUBADDRESS}:${GITHUBPROJECT}/${GITHUBREPO}
 
 
-#. Take results and create a "bad password" file. 
+#. Use ``tartufo`` to scan your repository and find any secrets in its history!
 
-   This file will be used by ``BFG`` and replace these flagged "bad password" entries with ``***REMOVED***``. It is important that you read through this file to make sure there are not exceptions that you want to remove and exclude with tartufo!
+   Use what you've learned to scan your repo!
```

Comment:
I'm still confused by this line. So far in these steps the user has basically only cloned their repo -- so what is it that they have learned at this point?

Reply:
the section is called "would you like to know more" in my opinion this is the example you would goto after you did the pre-requisite reading. if that's confusing I can take it out

Reply:
I think it might be -- just because, you can't control where a user is coming from. I would personally click on "Would you like to know more?" just out of sheer curiosity, and then stick around because it claims to be an "end-to-end example", which I would take to mean "all-inclusive".

Reply:
updated